circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -85,11 +85,16 @@ class CircularSeekBar @JvmOverloads constructor(
     private val circleProgressGlowPaint: Paint = Paint()
 
     /**
-     * `Paint` instance used to draw the center of the pointer.
+     * `Paint` instance used to draw the pointer.
      * Note: This is broken on 4.0+, as BlurMasks do not work with hardware acceleration.
      */
     private val pointerPaint: Paint = Paint()
 
+    /**
+     * `Paint` instance used to draw the pointer's fill. Will effectively use [pointerPaint] as stroke.
+     */
+    private val pointerFillPaint: Paint = Paint()
```

Comment:
It looks good to have an smaller circle inside the original pointer. To help user understanding this idea, do you think a name like "pointerInner" make more sense than "pointerFill"?

Which means, you may add two properties:

1. pointerInnerColor
2. pointerInnerStrokeWidth


Or you may also consider something like "pointerOverlay" to reflect the actually implementation is drawing an overlay on top of existing pointer.

Reply:
I actually really like `pointerOverlay` alot. Will make that change